frontend/src/components/PrivateRoute/PrivateRoute.js

```diff
@@ -13,6 +13,10 @@ class PrivateRoute extends Component {
                 case getMeStatus.SUCCESS:
                     if (this.props.history.location.pathname === "/") {
                         this.props.history.push("/main");
+                    } else if (this.props.history.location.pathname === "/init") {
+                        if (this.props.history.location.state == null || this.props.history.location.state !== "signup") {
```

Comment:
Wow, I think this can be used for choosing the default tab of pages, can't it?

Reply:
Yes I think so, too!